docusaurus.config.js

```diff
@@ -121,7 +121,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl: 'https://github.com/web3-api/documentation/tree/main',
+          editUrl: 'https://github.com/web3-api/documentation/tree/main/src',
```

Comment:
I think this should be 'https://github.com/polywrap/documentation'

Reply:
Aha, thanks. I also had to add `/tree/main/src` at the end for it to work correctly (locally).

Does it look right now?

Reply:
Yeah, it looks good